src/Classes/ServiceAPI/MyRadio_TrainingStatus.php

```diff
@@ -312,9 +312,9 @@ public static function getAllAwardableTo(MyRadio_User $to, MyRadio_User $by = nu
 
         $statuses = [];
         foreach (self::getAll() as $status) {
-            if ((!$status->isAwardedTo($to))
+            if ($status->canAward($by)
                 && $status->hasDependency($to)
-                && $status->canAward($by)
+                && (!$status->isAwardedTo($to))
```

Comment:
Explain?

Reply:
For some reason if you run canAward without the other tests it runs smoothly. Obviously I can't just remove the tests so I just put that one first so when that fails it doesn't test the other ones.

Reply:
Not sure quite how significant it is but compare https://ury.org.uk/myradio/Profile/view/?memberid=11207 to https://ury.org.uk/myradio-dev/Profile/view/?memberid=11207 (ignoring the unrelated errors)